modules/tests/src/test/scala/dev/profunktor/redis4cats/TestScenarios.scala

```diff
@@ -190,6 +195,13 @@ trait TestScenarios { self: FunSuite =>
       _ <- IO(assert(y.contains(2)))
       z <- cmd.zCount(testKey, ZRange(0, 1))
       _ <- IO(assert(z.contains(1)))
+      _ <- cmd.zAdd(otherTestKey, args = None, scoreWithValue1, scoreWithValue3)
+      zUnion <- cmd.zUnion(args = None, testKey, otherTestKey)
```

Comment:
I would add one more case with `args = Some(...)` as well, I think that'd be nice to have

Reply:
👍 Test added